stable/odoo/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: odoo
-version: 6.2.2
-appVersion: 11.0.20190415
+version: 6.2.3
```

As it is a major in the application version, we should bump to a major in the chart version:
```suggestion
version: 7.0.0
```